src/app/utils/utils.js

```diff
@@ -335,10 +340,11 @@ function getReqParams(query = {}) {
   const q = query.q || '';
   const sort = query.sort || '';
   const order = query.sort_direction || '';
+  const sortQuery = query.sort_scope || '';
```

Comment:
I don't like the introduction of this new variable because we already have `sort` and `order` and `sort_scope` is the same, but I do see why you did this.